shared/models.go

```diff
@@ -40,3 +44,56 @@ type Browser struct {
 type Token struct {
 	Secret string `json:"secret"`
 }
+
+type Manifest struct {
+	Items   ManifestItems `json:"items,omitempty"`
+	Version *int          `json:"version,omitempty"`
+}
+
+func (m Manifest) FilterByPath(paths mapset.Set) (result Manifest, err error) {
```

Comment:
Should the receiver type be a pointer for consistency and to avoid copying? Though in this case the cost of copying isn't really a problem as all children types are mostly pointers. Yet using a pointer receiver type for struct seems to be a more consistent default. https://github.com/golang/go/wiki/CodeReviewComments#receiver-type

BTW, perhaps we can simply modify in place in this method, but I don't really have a preference for that.

Reply:
Done (modified in-place by copying the input).